README.md

```diff
@@ -115,10 +118,6 @@ The following VCF fields can be accessed in the filter expression:
   ```sh
   vembrane filter 'INFO["MQ"] == 60' variants.bcf
   ```
-* Only keep annotations and variants where consequence contains the word "stream" (matching "upstream" and "downstream"):
-  ```sh
-  vembrane filter 're.search("(up|down)stream", ANN["Consequence"])' variants.vcf
-  ```
```

Comment:
Yes, I wanted to find a replacement for this (using a regex) but didn't come up with a good example, yet.
This is outdated, because Consequence used to be `str`, but is now `set[str]` as it should be.

Reply:
OK, so I'll leave this unresolved for now, as a reminder. Should you eventually decide against giving another regex example, just resolve this anyways...